shard-operator/src/test/java/com/redhat/service/bridge/shard/operator/ManagerSyncServiceTest.java

```diff
@@ -55,12 +55,13 @@ public void setup() {
     @WithPrometheus
     public void testBridgesAreDeployed() throws JsonProcessingException, InterruptedException {
         List<BridgeDTO> bridgeDTOS = new ArrayList<>();
-        bridgeDTOS.add(new BridgeDTO("bridgesDeployed-1", "myName-1", "myEndpoint", TestSupport.CUSTOMER_ID, ManagedResourceStatus.ACCEPTED, TestSupport.KAFKA_CONNECTION_DTO));
-        bridgeDTOS.add(new BridgeDTO("bridgesDeployed-2", "myName-2", "myEndpoint", TestSupport.CUSTOMER_ID, ManagedResourceStatus.ACCEPTED, TestSupport.KAFKA_CONNECTION_DTO));
+        bridgeDTOS.add(new BridgeDTO("bridgesDeployed-1", "myName-1", "myEndpoint/events", TestSupport.CUSTOMER_ID, ManagedResourceStatus.ACCEPTED, TestSupport.KAFKA_CONNECTION_DTO));
```

Comment:
This change in the test is not really necessary; it is in essence mocked/stubbed DTOs/JSON for `WireMockServer`.